config/initializers/devise.rb

```diff
@@ -98,31 +73,6 @@
   # Setup a pepper to generate the encrypted password.
   # config.pepper = 'd68602c07af77ffeb4817fad17efeda6ece23b89d3b2b42126b56b2a99a8cf5c653f47ad1298a1b39d8e866835b48148875e84883c308c3addf4d3acbc124e60'
 
-  # ==> Configuration for :confirmable
```

Comment:
We aren't using confirmable because responsibility for authentication lies in our district partners' LDAP systems.